index.html

```diff
@@ -22,18 +61,24 @@
     <form id="calc" onSubmit="return calc();">
       <fieldset form="calc">
         <legend>Input</legend>
-        <label for="mac">MAC address</label>
-        <input name="mac" id="mac" placeholder="de:ad:be:ef:00:00"/>
-        <button type="submit">Calculate</button>
+        <span class="input">
+          <label for="mac">MAC address</label>
+          <input name="mac" id="mac" placeholder="de:ad:be:ef:00:00"/>
+          <button type="submit">Calculate</button>
+        </span>
       </fieldset>
       <fieldset form="calc">
         <legend>Output</legend>
-        <label for="mesh">Mesh IP</label>
-        <input name="mesh" id="mesh" disabled="disabled"/>
-        <button id="copy-mesh" type="button">copy</button>
-        <label for="wlan">WLAN IP</label>
-        <input name="wlan" id="wlan" disabled="disabled"/>
-        <button id="copy-wlan" type="button">copy</button>
+        <span class="output">
```

Comment:
Yes, use inline display.

--
Colin Dean
http://cad.cx



On Mar 28, 2016, 09:43, at 09:43, Jason Khanlar <notifications@github.com> wrote:
>>        </fieldset>
>>        <fieldset form="calc">
>>          <legend>Output</legend>
>> -        <label for="mesh">Mesh IP</label>
>> -        <input name="mesh" id="mesh" disabled="disabled"/>
>> -        <button id="copy-mesh" type="button">copy</button>
>> -        <label for="wlan">WLAN IP</label>
>> -        <input name="wlan" id="wlan" disabled="disabled"/>
>> -        <button id="copy-wlan" type="button">copy</button>
>> +        <span class="output">
>
>span for display inline, changed to block < 400px. Is it better to use
>div and apply display: inline?
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/pittmesh/ip-calculator/pull/18/files/f33be2429b2081e5dfde38cdfb4475ed0e853764..3b5664fb20672fe7ce7d546e024f3a7ab0a23cfa#r57572224